src/pages/main.js

```diff
@@ -43,6 +43,7 @@ const Main = (props) => {
           {!comparisonMode && <Map />}
         </div>
         {props.selectedFeature && !comparisonMode && <Legend absolute />}
+        <Footer inDarkMode/>
```

Comment:
If you make `inDarkMode={!comparisonMode}`, it should update at the right times.